### PR TITLE
Add documentation for publishing Fusion messages

### PR DESCRIPTION
**Pull request by hans-notifi**

### CODE REVIEW

File: docs/alert-trigger/node-client/index.md [hunk 1]
```diff
@@ -1,90 +1,93 @@
...
-To follow along, please clone the sample of the Node Client on
-[Github](https://github.com/notifi-network/notifi-sdk-ts/tree/main/packages/notifi-node-sample).
+## Prerequisites
 
-First, start the server with `npm run dev`.
+In order to use the Node SDK to publish notifications, you first need to set up the following on the
+[Notifi Admin Portal](https://admin.notifi.network):
 
-## login
+- A Dapp account (an account created on this page, NOT the Notifi Hub)
```
Comment (kai-notifi) [inline docs/alert-trigger/node-client/index.md]: "this page" to Admin Portal

File: docs/alert-trigger/node-client/index.md [hunk 2]
```diff
@@ -1,90 +1,93 @@
...
+- A "Sent from my server" topic, created in Alert Manager
```
Comment (kai-notifi) [inline docs/alert-trigger/node-client/index.md]: Would be nice to link to the topic creation for this

**Review by kai-notifi — APPROVED** [review]

**Review by nimeshamin — APPROVED** [review]

**Comment by nimeshamin** [comment]
@hans-notifi 
Docs markdown link couldn't be resolved: (../../create-topics/templates.md) in "/home/runner/work/notifi-network.github.io/notifi-network.github.io/docs/alert-trigger/node-client/index.md" for version current

**Comment by hans-notifi** [comment]
> @hans-notifi Docs markdown link couldn't be resolved: (../../create-topics/templates.md) in "/home/runner/work/notifi-network.github.io/notifi-network.github.io/docs/alert-trigger/node-client/index.md" for version current

That link is for a page created in here: https://github.com/notifi-network/notifi-network.github.io/pull/15

**Pull request merged**